### DISCUSSION TITLE
Unit in GHG FBS

### DISCUSSION THREAD

**Discussion by MoLi7** [USEPA/flowsa#424] [Q&A] [answered]
Hi @bl-young , could you confirm if the unit in `GHG_national_2022_m2_v2.0.3_1cb504c` is **kg CO2e** or **kg of each GHG**? 

I thought it was the latter, because in _useeior_ there is a step to characterize GHG, i.e. `C %*% B`. However, the [GHG methods](https://github.com/USEPA/flowsa/blob/v2.0.3/flowsa/methods/flowbysectormethods/GHG_national_m2_common.yaml) indicate [GHGI Table 3-8 and 3-9](https://www.epa.gov/system/files/documents/2024-04/us-ghg-inventory-2024-main-text_04-18-2024.pdf#page=155) are used, where the unit is kg CO2e. Now I'm quite confused and would appreciate some clarification. 🙏 

Best,
Mo

**Comment by bl-young** [comment, marked as answer]
It is in kg, except for the HFCs and PFCs unspecified flow which has to remain in kg CO2e. 

You are right that the original tables are often in CO2e, but we convert back to the raw gas total with a [mapping file](https://github.com/USEPA/flowsa/blob/master/flowsa/methods/flowbysectormethods/GHG_national_m1_common.yaml#L55) during Flow by Sector creation.

**Reply by MoLi7** [reply]
Thank you so much Ben for your prompt reply. That additional mapping to un-characterize makes sense.
